src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java

```diff
@@ -18,20 +19,35 @@
 import java.util.stream.IntStream;
 
 /**
- * Given pileup data from {@link GetPileupSummaries} calculate the fraction of reads coming from exogenous human DNA.
+ * Given pileup data from {@link GetPileupSummaries}, calculates the fraction of reads coming from cross-sample contamination.
  *
- * This tool is run as follows:
+ * <p>
+ *     The resulting contamination table is used with either {@link org.broadinstitute.hellbender.tools.walkers.mutect.Mutect2}
```

Comment:
@sooheelee Excellent point, however these are relics of shared HaplotypeCaller code.  Mutect2 doesn't do anything with these arguments.  This probably means I should move them out of shared code and into HaplotypeCaller-only code.